.gitlab-ci.yml

```diff
@@ -525,3 +502,54 @@ deploy-master-to-dev:
   - helm upgrade magda deploy/helm/magda --install --recreate-pods -f deploy/helm/magda-dev.yml --set global.image.repository=registry.gitlab.com/magda-data/magda/data61,global.image.tag=master --timeout 1200 --wait
   - kubectl delete configmap connector-config --ignore-not-found && kubectl create configmap connector-config --from-file deploy/connector-config/
   - kubectl apply -f deploy/kubernetes/generated/prod/cron
+
+Release Tags To Docker Hub:
+  stage: release
+  only:
+    - /^v.*$/
+  except:
+    - branches
+    - triggers
+  dependencies:
+    - builders-and-yarn
+  cache: {}
+  services:
+    - docker:dind
+  image: registry.gitlab.com/magda-data/magda/data61/magda-builder-docker:$CI_COMMIT_REF_SLUG
+  retry: 1
+  script:
+    - docker login -u gitlab-ci-token -p $CI_JOB_TOKEN $CI_REGISTRY
+    - docker login -u magdabot -p $DOCKER_HUB_PASSWORD
+    - yarn install
+    - yarn run retag-and-push -- -- --fromPrefix=registry.gitlab.com/magda-data/magda/ --fromVersion=$CI_COMMIT_REF_SLUG
```

Comment:
Not sure the env where the script will be run but do we need to add an entry to `scripts` section of the `package.json` so that `yarn` can recognize command `etag-and-push`?

Reply:
Oh dear, looks like it disappeared in a merge :(

Reply:
... all of them too :'(